src/Components/Upcoming/Upcoming.tsx

```diff
@@ -318,14 +336,14 @@ const UpcomingTab: React.FC<React.PropsWithChildren> = () => {
               >
                 <CardTitle className="drf-lifecycle__upcoming-card">
                   <InfoCircleIcon color={'#2B9AF3'} />
-                  Upcoming additions
+                  Upcoming additions and enhancements
                 </CardTitle>
               </CardHeader>
               <CardBody>
                 <span className="drf-lifecycle__upcoming-count">
                   {numAdditions}
                 </span>{' '}
-                upcoming {pluralize(numAdditions, 'addition')} that could affect
+                upcoming {pluralize(numAdditions, 'addition')} and/or {pluralize(numAdditions, 'enhancement')} that could affect
```

Comment:
In the mocks I see there are just additions in the title:
![Screenshot From 2025-04-23 17-55-25](https://github.com/user-attachments/assets/8c44ff4b-45f0-4f08-9712-ffc158c424cc)
But I believe this is a fine change as I see there is a new type of upcoming features.

Reply:
Yes this was from a separate conversion with Brian and John about updating our UX. We're also removing references to Upcoming in the Roadmap tab